<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QDialog" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1432</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> Hotel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/projetimages/projetimages/e.png</normaloff>:/projetimages/projetimages/e.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
 selection-background-color:red;
border-radius:15px;
}
QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}
QLineEdit#cherclientutil_txt
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}
QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QDateEdit {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QLineEdit
{

    border: 1px solid gray;
    border-radius: 4px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}
QDialog#Client
{
	background-image:url(:/projetimages/projetimages/a.jpg);

}
*{
font-family:centry gothlic ;
font-size:15px;
}
#frame
{

	background:rgba(0,0,0,0.7);
border-radius:15px;
}


QPushButton
 {
     background-color: red;

     border-radius: 10px;

 }

#label{
color:white;
background:transparent;

}

#label_2
{
color:white;
background:transparent;

}


#label_3{
color:white;
background:transparent;

}

#label_4{
color:white;
background:transparent;

}

#label_5{
color:white;
background:transparent;

}

#label_6{
color:white;
background:transparent;

}


#label_7{
color:white;
background:transparent;

}



QPushButton
{
color:white;

border-raduis:15px;
}
QPushButton#modifierc_btn:hover
{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}
QPushButton#supprimer_btn:hover
{
color: white;
border-radius:15px;
background:rgb(255, 85, 127);
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>1371</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>180</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CIN :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>250</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date de naissance : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifierc_btn">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Télephone :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>310</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="adressec_txt">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>310</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="datenaissancec_txt">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>250</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableViewC">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>1011</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cinc_txt">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>120</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénom :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prenomc_txt">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>110</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>60</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nomc_txt">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>60</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="telephonec_txt">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>380</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cherclientutil_txt">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Saisir le CIN du client à chercher :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer_btn">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
